### DISCUSSION TITLE
Long time no see .... stutter party is back

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#7110] [General]
Hi simmers and developers,

today I flew a leg round about 3,5 h. After 3 hours I got a long time missed stutter part ... :-( .... same setup as every time. Does someone else have problemes with frame drops right now?

![grafik](https://user-images.githubusercontent.com/76531893/165288140-cc5da553-dc25-4300-9564-630f8ecfc2c4.png)

When switching to exterior view it will calm down and be smooth ... 

![grafik](https://user-images.githubusercontent.com/76531893/165288835-99005688-4749-4296-ad20-02f1fc5a950e.png)

hui ..... hellfire :-p ...

![grafik](https://user-images.githubusercontent.com/76531893/165294993-2627b33b-70e0-464d-8689-2504df31298b.png)

greets

**Comment by aguther** [comment]
try to turn off nav & logo lights in that case and see if it helps.

if you're using YourControls remove it when not used.

and no: this is not a joke. surprisingly, it helped some recently. reason is unclear.

**Reply by CptBayer** [reply]
Cool ... I will remeber your hint :-p ...

greets klaus
